src/components/NewGalaxyModal.js

```diff
@@ -30,6 +30,8 @@ export const NewGalaxyModal = withModalDrawer({ width: 675 })(class NewGalaxyMod
 
   constructor(props) {
     super(props)
+    const { apps } = props
```

Comment:
Do we need this here? Or is this more so we have a list of props here? (I'm worried it will fire unused variable errors)